src/main/java/seedu/modtracker/Module.java

```diff
@@ -8,24 +8,26 @@
  */
 public class Module {
 
-    private String moduleCode;
-    private int expected = -1;
-    private double[] actualTime = new double[13];
+    public static final int NO_INPUT = -1;
+    public static final int INDEX_OFFSET = 1;
+    private final String moduleCode;
```

Comment:
I think the moduleCode should not be set as final as we would want the moduleCode to change.

Reply:
Okay sure will change it back